rn/Teacher/ios/Teacher/Submissions/QuizSubmissionList/View/QuizSubmissionListView.swift

```diff
@@ -0,0 +1,159 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2023-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+import Core
+
+public struct QuizSubmissionListView: View {
+    @ObservedObject private var model: QuizSubmissionListViewModel
+    @Environment(\.viewController) private var controller
+
+    init(model: QuizSubmissionListViewModel) {
+        self.model = model
+    }
+
+    public var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            filterBarView
+            Color.borderMedium
+                .frame(height: 0.5)
+            if model.state == .loading {
+                loadingIndicator
+            } else {
+                List {
+                    switch model.state {
+                    case .data:
+                        submissionList
+                    case .empty:
+                        EmptyPanda(.Teacher,
```

Comment:
Use `InteractivePanda` instead. `ConferencesPanda` scene has the same teacher image.